### PR TITLE
Issue12 product page listing part3 maps styling

### PR DESCRIPTION
**Pull request by ninetteadhikari**
- Adds a map with markers that update according to the filters selected
- Material UI styling files

### CODE REVIEW

File: client/src/components/ProductsList.js [hunk 1]
```diff
@@ -65,79 +76,113 @@ export default class ProductsList extends Component {
...
-    const filteredProduct = this.state.products.filter(product => {
+    const distinctBrand = [
+      ...new Set(
+        this.state.products.map(product => {
+          return product.brand;
+        })
+      )
+    ];
+
+    const maxPrice = Math.max(
```
Comment (1000miles) [inline client/src/components/ProductsList.js:88]: Is it possible to provide a max price range 0 - 20.000 instead of max divided through products?

**Review by 1000miles — COMMENTED** [review]
Thanks for the PR, @ninetteadhikari. I left a comment.

File: client/src/components/GoogleMapsProductList.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        >
+          <Paper>
+            <Typography variant="headline" component="h4">
+              Changing Colors Garage
+            </Typography>
+            <Typography component="p">
+              98G Albe Dr Newark, DE 19702 <br />
+              302-293-8627
+            </Typography>
+          </Paper>
```
Comment (Taliko5) [inline client/src/components/GoogleMapsProductList.js:81]: adding a marker

**Comment by ninetteadhikari** [comment]
Have decided to switch to MapBox. So closing this PR and will raise a new one with Mapbox codes

**Pull request closed without merging**